src/common/balances.lua

```diff
@@ -60,7 +62,7 @@ end
 ---@return table<string, string>
 function balances.setDescription(description)
 	assert(type(description) == "string", "Description must be a string")
-	assert(#description <= 512, "Description must not be longer than 512 characters")
+	assert(#description <= constants.MAX_DESCRIPTION_LENGTH, "Description must not be longer than " .. constants.MAX_DESCRIPTION_LENGTH .. " characters")
```

Comment:
same question here

Reply:
Well for this one we were following the previous pattern of having the metadata utils here.